apps/projects/app/components/Panel/NewIssue/NewIssue.js

```diff
@@ -103,66 +112,125 @@ class NewIssue extends React.PureComponent {
 
     const id = selectedProject > 0 ? reposIds[selectedProject - 1] : ''
 
+    const handleSubmit = async e => {
+      e.preventDefault()
+      this.props.closePanel()
+      const newIssueData = {
+        number: issues.filter(i => i.data.repository && (i.data.repository.hexId === repoHexIds[selectedProject - 1])).length,
+        id: repoHexIds[selectedProject - 1] + '_' + issues.filter(i => i.data.repository && (i.data.repository.hexId === repoHexIds[selectedProject - 1])).length,
+        title,
+        body: description,
+        author: {
+          login: this.props.account,
+          avatarUrl: '',
+          url: '',
+        },
+        repository: reposManaged[selectedProject - 1],
+        labels: { totalCount: 0, edges: Array(0) },
+        milestone: null,
```

Comment:
It could make sense to get rid of the `milestone` field if we don't want to use it anymore, it was removed from the filterbar but still present across the code, and apparently always being assigned a "null" value, what is the purpose of having a value that will always be null? This may happen for other fields here.